proto/process/agent.proto

```diff
@@ -603,6 +611,20 @@ message DNSEntry {
   repeated string names = 1;
 }
 
+message Cluster {
+  int32 nodeCount = 3;
+  map<string, int32> kubeletVersions = 4;
+  map<string, int32> apiServerVersion = 5;
+  // sum of all nodes capacities and allocatables
+  uint32 podCapacity = 6;
+  uint32 podAllocatable = 7;
+  uint64 memoryAllocatable = 8;
```

Comment:
in which unit will this be?

Reply:
the unit will be the smallest possible unit. Example bytes for memory and millicores for cpu

Reply:
could we add this add a comment? 🙏

Reply:
you are right, I added this comment as well 👍